docs/spfx/library-component-tutorial.md

```diff
@@ -120,20 +129,6 @@ localization_priority: Priority
 1. Deploy this package in the tenant app catalog and make it tenant wide deployed by checking the ```Make this solution available to all sites in the organization``` option.
```

Comment:
Incorrect formatting... not code, use bold formatting.

Reply:
Updated to bold